browser_tests/tests/curveWidget.spec.ts

```
@@ -0,0 +1,72 @@
+import { expect } from '@playwright/test'
+
+import { comfyPageFixture as test } from '../fixtures/ComfyPage'
```

ESLint

[error] '../fixtures/ComfyPage' import is restricted from being used by a pattern. Use the `@e2e/` path alias instead of relative imports.

(no-restricted-imports)